### PR TITLE
Team Page API Fetching

### PR DESCRIPTION
**Pull request by Hgjnnf**
- Using axios to fetch data with deployed API

### CODE REVIEW

File: src/components/team/Team.jsx [hunk 1]
```diff
@@ -1,96 +1,55 @@
...
+import { client } from "../utils/apiClient";
 import Rachit from "../../assets/team_pictures/rachit.png";
 import Rithvik from "../../assets/team_pictures/rithvik.png";
-import Grace from "../../assets/team_pictures/grace.jpeg";
-import Gunyup from "../../assets/team_pictures/gunyup.png";
+
+async function getExecs() {
+  try {
+    const response = await client.get('/executive')
+    return JSON.parse(response.data.slice(0,-4))
```
Comment (IcedTea2K) [inline src/components/team/Team.jsx:11]: Just wondering, what are we excluding with the `data.slice()` here?

Comment (IcedTea2K) [reply src/components/team/Team.jsx:11]: Besides this, everything else looks good to merge

Comment (Hgjnnf) [reply src/components/team/Team.jsx:11]: null, which is what I brought up cuz the api is return null at the end.

Comment (Hgjnnf) [reply src/components/team/Team.jsx:11]: > Besides this, everything else looks good to merge

Make sure to approve it, so I can merge. It's currently blocked <3

Comment (IcedTea2K) [reply src/components/team/Team.jsx:11]: > null, which is what I brought up cuz the api is return null at the end.

Ah okay, that makes sense. And yeah, my bad for forgetting to approve it :)

**Review by IcedTea2K — APPROVED** [review]
LGTM! 👍

**Pull request merged**